spec.md

```diff
@@ -19,10 +20,72 @@ for public re-use.
                               # with dimension order (t, c, z, y, x).
 ```
 
+## "multiscales" metadata
+
+Metadata about the multiple resolution representations of the image can be
+found under the "multiscales" key in the group-level metadata.
+The specification for the multiscale (i.e. "resolution") metadata is provided
+in [zarr-specs#50](https://github.com/zarr-developers/zarr-specs/issues/50).
+If only one multiscale is provided, use it. Otherwise, the user can choose by
+name, using the first multiscale as a fallback:
+
+```
+datasets = []
+for named in multiscales:
+    if named['name'] == '3D':
+        datasets = [x['path'] for x in named["datasets"]]
+        break
+if not datasets:
+    # Use the first by default. Or perhaps choose based on chunk size.
+    datasets = [x['path'] for x in multiscales[0]["datasets"]]
+```
+
+The subresolutions in each multiscale are ordered from highest-resolution
+to lowest.
 
 see also:
  - [Multiscale metadata spec](https://github.com/zarr-developers/zarr-specs/issues/50)
 
-| Revision | Date       | Description                              |
-|----------|------------|------------------------------------------|
-| 0.1      | 2020-04-20 | First version for internal demo          |
+##  "omero" metadata
+
+Information specific to the channels of an image and how to render it
+can be found under the "omero" key in the group-level metadata:
+
+```
+"id": 1,                              # ID in OMERO
+"name": "example.tif",                # Name as shown in the UI
+"channels": [                         # Array matching the c dimension size
+    {
+        "active": true,
+        "coefficient": 1,
+        "color": "0000FF",
+        "emissionWave": null,
```

Comment:
Don't think we need emissionWave - it was used for channel label but is now redundant.

Reply:
Done (on minio-dev as well)